packages/driver/test/cypress/integration/commands/actions/click_spec.js

```diff
@@ -25,42 +29,12 @@ const mouseHoverEvents = [
 ]
 const focusEvents = ['focus', 'focusin']
 
-const attachListeners = (listenerArr) => {
-  return (els) => {
-    _.each(els, (el, elName) => {
-      return listenerArr.forEach((evtName) => {
-        el.on(evtName, cy.stub().as(`${elName}:${evtName}`))
-      })
-    })
-  }
-}
-
```

Comment:
I like that this was pulled out into `support/utils`. Are there any other places in tests that do similar logic that can be replaced with `support/utils`? The rest of the comment is predicated on the assumption that there are additional places that could utilize this functionality. If there are not, then there's nothing to see here. ;-)

I ask because when I (any dev really) come back and reference one of these specs to use as an example to create a new, yet similar spec, it would be ideal if the pattern is using `support/utils`; instead of having potentially two ways do it.

Reply:
similar logic can be moved here, sure